config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "dishes/:id", to: "dishes#show"
+  patch "dishes/:id", to: "dishes#update"
```

Comment:
Should be: `post "dishes/:id/ingredients", to: "ingredients#create"` or `post "dishes/:id/ingredients", to: "dish_ingredients#create"`